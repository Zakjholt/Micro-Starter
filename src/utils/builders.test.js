// By default, our test script selects all files ending in _test.js.
const test = require('ava')
const { omit } = require('lodash')
const { statusBuilder, echoBuilder } = require('./builders')

test('statusBuilder', t => {
  const payload = statusBuilder()

  t.snapshot(payload)
})

test('echoBuilder', t => {
  const payload = echoBuilder({ message: 'These ones go to eleven' })
  // Snapshot testing will fail with properties that change, like dates, so we omit them
  const safeSnapPayload = omit(payload, 'date')

  t.snapshot(safeSnapPayload)
})
