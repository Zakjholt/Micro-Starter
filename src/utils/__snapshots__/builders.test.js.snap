// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`echoBuilder 1`] = `
Object {
  "message": "These ones go to eleven",
}
`;

exports[`statusBuilder 1`] = `
Object {
  "status": "healthy",
}
`;
